configs/app/features/marketplace.ts

```diff
@@ -7,21 +7,31 @@ import { getEnvValue, getExternalAssetFilePath } from '../utils';
 const configUrl = getExternalAssetFilePath('NEXT_PUBLIC_MARKETPLACE_CONFIG_URL');
 const submitFormUrl = getEnvValue('NEXT_PUBLIC_MARKETPLACE_SUBMIT_FORM');
 const categoriesUrl = getExternalAssetFilePath('NEXT_PUBLIC_MARKETPLACE_CATEGORIES_URL');
+const adminServiceApiHost = getEnvValue('NEXT_PUBLIC_ADMIN_SERVICE_API_HOST');
 
 const title = 'Marketplace';
 
-const config: Feature<{ configUrl: string; submitFormUrl: string; categoriesUrl: string | undefined }> = (() => {
+const config: Feature<{
+  configUrl: string | undefined;
+  submitFormUrl: string;
+  categoriesUrl: string | undefined;
+  api: { endpoint: string; basePath: string } | undefined;
+}> = (() => {
```

Comment:
Since the `configUrl` and `api` properties are mutually exclusive, I would suggest the following structure of the config:

```js
type Config = (
  { configUrl: string } | 
  { api: { endpoint: string; basePath: string } }
) & { submitFormUrl: string; categoriesUrl: string | undefined }
const config: Feature<Config> = ...
```